cnf-certification-test/networking/netutil/netutil.go

```diff
@@ -77,3 +77,13 @@ func GetListeningPorts(cut *provider.Container) (map[PortInfo]bool, error) {
 
 	return parseListeningPorts(outStr)
 }
+
+func GetSSHDaemonPort(cut *provider.Container) (string, error) {
+	const findSSHDaemonPort = "ss -tpln | grep sshd | head -1 | awk '{ print $4 }' | awk -F : '{ print $2 }'"
+	outStr, errStr, err := crclient.ExecCommandContainerNSEnter(findSSHDaemonPort, cut)
+	if err != nil || errStr != "" {
+		return "", fmt.Errorf("failed to execute command %s on %s, err: %v", findSSHDaemonPort, cut, err)
+	}
```

Comment:
```suggestion
	if err != nil {
		return "", fmt.Errorf("Failed to execute command %s on %s, err code: %v, err str: %s.", findSSHDaemonPort, cut, err, errStr)
	}
```

Reply:
That condition is present in the repo in multiple places from ExecCommandContainer. I don't know what case would give no error, but errStr, but I think there may be some case. Hence, that condition was with OR.